src/components/NavbarTemp.vue

```diff
@@ -84,50 +84,46 @@ li b-button:hover {
     background-color: #111;
 }
 /*end of left column*/
-#ulcent{
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-  overflow: hidden;
-  overflow-y: scroll;
-  background-color: #30aef2;
+#ulcent {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    overflow-y: scroll;
+    background-color: #30aef2;
 }
 
 #licent {
-  float: left;
+    float: left;
 }
 
 #butcent {
-  display: block;
-  color: rgb(0, 0, 0);
-  margin: 0;
-  text-align: center;
-  padding-top: 18px;
-  padding-right: 20px;
-  padding-bottom: 20px;
-  padding-left: 20px;
-  text-decoration: none;
+    display: block;

```

Comment:
conflict with main branch need to be resolved

Reply:
Fix conflicts